compiler/rustc_macros/src/lib.rs

```diff
@@ -163,6 +163,7 @@ decl_derive!(
 decl_derive!(
     [SessionSubdiagnostic, attributes(
         // struct/variant attributes
+        diag,
```

Comment:
Why is this necessary?

Reply:
It's not, I forgot to remove it after my spaghetti recipe